sub-packages/bionemo-esm2/src/bionemo/esm2/run/recipes.py

```diff
@@ -370,7 +371,7 @@ def esm2_tiny_test_recipe(args):
 
 def main():  # noqa: D103
     def parse_args():
-        parser = argparse.ArgumentParser(description="Create ESM2 configuration JSON.")
+        parser = argparse.ArgumentParser(description="Create ESM2 configuration YAML.")
         parser.add_argument(
```

Comment:
instead of listing recipes, we can use some pydantic class RecipeType for ESM2  choices=RecipeTypePretrain[ESM2].__fields__.keys() to have pydantic as validator

Reply:
ooooo interesting, I've never done this. I'll give it a try.

Reply:
Done